lifecycle-operator/chart/README.md

```diff
@@ -0,0 +1,86 @@
+# Keptn
+
+Keptn introduces a more cloud-native approach for pre- and post-deployment,
+as well as the concept of application health checks
```

Comment:
```suggestion
and application health checks
```
The concept of application health checks is out there -- we are offering an implementation, right?  Should this also mention that you get the pre/post-deployment and health checks for all the workloads in your Application?